lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -249,7 +249,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_belarus
 
-      outcome :outcome_os_italy
+      outcome :outcome_opposite_sex_marriage_italy_residing_in_uk_or_italy
```

Comment:
The naming of the outcome seems inconsistent compared to https://github.com/alphagov/smart-answers/pull/2389/commits/5dff9652a71a46e99620717addff921ef5e553f4 i.e. `uk_or_ceremony_country` vs `uk_or_italy`.